conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+            </ul>
+
+            <section id="professional-mediation">
+                <h3>Professional Mediation</h3>
+
+                <p>In cases where it's deemed necessary to bring in a professional mediator, all parties involved:</p>
+
+                <ul>
+                    <li>will agree to attend mediation as an option for resolving the issue</li>
+                    <li>will individually attend a meeting with independent trained mediators</li>
+                    <li>will collectively attend a meeting to state how each party sees the problem and how it might be resolved</li>
+                    <li>will engage with the mediator who will work through the issues confidentially and confirm agreements with everyone as they are reached</li>
+                </ul>
+
+                <p>TODO: Details of how to access mediation</p>
+            </section>
+        </section>
+
+        <section id="investigations">
+            <h2>Investigations</h2>
+
+            <p>If resolution attempts have not worked or the issue is not appropriate for mediation, a participant can raise a formal complaint against another participant.</p>
+
+            <p>TODO: Outline where and how to send a formal complaint.</p>
+
+            <p>A formal complaint should be in writing, and include the following:</p>
+
+            <ul>
+                <li>the grounds for the complaint (for example, what parts of the CoC you think are relevant)</li>
+                <li>if appropriate, what you have done so far to try to resolve the complaint (for example, trying to solve the issue yourself or through mediation)</li>
+                <li>any evidence or applicable materials relating to the conflict (e.g., emails, minutes, screenshots, or other materials)</li>
+                <li>what resolution or action you wish to be taken to address the conflict</li>
+            </ul>
+
+            <p>An investigator will be appointed and will talk to both parties and anyone else relevant. The investigator's role is to:</p>
+
+            <ul>
+                <li>establish what happened, in a fair and objective manner</li>
+                <li>ensure that the investigation is proportionate to the seriousness and complexity of the complaint</li>
+                <li>ensure that the investigation is concluded without delay</li>
+                <li>keep notes and submitted evidence in confidence</li>
+            </ul>
+
+            <p>The investigator will write a summary of the result of the investigation and recommend what actions should be taken. Outcomes and actions will be agreed by W3M and communicated to both parties.</p>
```

Comment:
The first sentence in the line above is also about what the investigator is supposed to do, so it should be included in the above bullets instead of being a separate sentence:

```
<li>write a summary of the result of the investigation</li>
<li>recommend what actions should be taken</li>
```